draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -466,8 +472,8 @@ authorized(AS x, AS y) =  / Else, "Provider+" if the U-SPAS entry
           If multiple eBGP sessions can segregate the Complex peering relationship into eBGP sessions with normal peering relationships the receiving/verifying AS SHOULD select the algorithm (per <xref target="Upflow"/> or <xref target="Downflow"/>) for each of the normal sessions based on its peering relation type.
         </t>
         <t>
-          If Complex peering relation can't be segregated (i.e., when a Complex BGP relationship occurs within one single BGP session), an operator may want to achieve an equivalent outcome by configuring policies on a per-prefix basis corresponding to the peering relation for the prefix.
-          If this option is not feasible, the operator MAY apply <xref target="Downflow"/>.
+          If a Complex peering relation cannot be segregated (i.e., when a Complex BGP relationship occurs within one single BGP session), an operator may want to achieve an equivalent outcome by applying an appropriate algorithm (<xref target="Upflow"/> or <xref target="Downflow"/>) on a per-prefix basis corresponding to the peering relation for the prefix.
```

Comment:
@mitradir  
Where is the mistake? My wording does not restrict it to one procedure. It says that "... applying an appropriate algorithm (Section 7.2 or Section 7.3)...".  Aren't these two sections the upflow and downflow procedures, respectively?

Reply:
@mitradir 
v-19 text says "...such attacks might go undetected by its customers or upper providers." But the truth is that "hijacked prefixes with forged-origin or forged-path-segment" are detected by upper providers. Only customers cannot detect that.  I feel that we do not need to mention other path manipulations here without saying what they are.  Anyway, in the next section (9.3), we do talk about other path manipulations in the form of altering AS prepends.

I changed the section title (9.2) to "Attack by a Provider" from "Provider as Trusted Point" because it is not true that the provider is always considered as a trusted point by a customer in the draft procedures. Just that the customer is not able to detect some forms of attacks by the provider while it can detect other forms of attacks.

Reply:
Can I explain why you deleted:
If this option is not feasible, the operator MAY apply <xref target="Downflow"/>.